src/Json.php

```diff
@@ -57,27 +57,27 @@ public function getEndpoint()
         return $this->endpoint;
     }
 
-    public function get($path, array $params = [])
+    public function get($path, array $params = [], array $headers = [])
```

Comment:
FYI, I noticed that I'd forgotten to fix the interface variable names (and their description), so I tweaked that with one of the latest commits.